src/app/components/SubmitButton/SubmitButton.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import SubmitButton from './SubmitButton';
+
+export default {
+  title: 'Component/SubmitButton',
+  Component: SubmitButton,
+};
+
+export const Signup = (): JSX.Element => (
+  <SubmitButton>
+    <img src="src/assets/SVGs/Signup.svg" />
+    Anmelden
+  </SubmitButton>
+);
```

Comment:
All these stories look very similar and more complicated than they need to be. I think it would be easier in the long run to solve it similar to how we did it in [moviescout](https://github.com/marwinburesch/moviescout/blob/main/src/app/components/MovieButton/MovieButton.tsx). This goes hand in hand with how SVG assets are used as well. If you have them as Reract components they get cleaner to call and more versatile in the end, due to the props. [Like this](https://github.com/marwinburesch/moviescout/blob/main/src/app/components/assets/BookmarkIcon.tsx)

Reply:
Is this "yes" as an anwer to my aformentioned question in the PR comment? :D